### ISSUE TITLE
[Keyboard navigation - Make code - Share project]: Keyboard focus indicator is not visible on the 'Experiments' button while navigating using tab key.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#5480]
### User Experience:

Users who depend on keyboard for navigation will get impacted if focus is not visible completely on the control. As a result, they will not be able to know about the exact focus of the keyboard while navigating the page.

### Note:

User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL : https://makecode.microbit.org/.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "More settings" button and invoke it.
6. Select 'About' menu item. Navigate the about dialog using tab key.
7. Observe whether keyboard focus indicator appears on the 'Experiments' button or not.

### Actual Result:
Issue: Keyboard focus indicator is not visible on the 'Experiments' button while navigating using tab key.

### Expected Result:

Rectangular keyboard focus indicator should be visible on the 'Experiments' button while navigating using tab key.

### MAS Reference:

[MAS 2.4.7 – Focus Visible](https://aka.ms/MAS2.4.7)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:

NA

### UIA/DOM details available:

NA

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- URL: https://makecode.microbit.org/.
<img width="968" alt="MAS2 4 7_Keyboard focus indicator is not visible on the 'Experiments' button while navigating using tab key" src="https://github.com/microsoft/pxt-microbit/assets/158465724/bc125191-0141-4bfa-a702-84b10314ae20">

**Comment by vroopea** [comment]
#Accessibility;#A11yMAS;#A11ySev2;#A11yAuto;#Tabstops;#K4W;#MAS2.4.7;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Jan2024;
